.github/workflows/ios-release-to-org-openfoodfacts-scanner.yml

```diff
@@ -41,7 +41,7 @@ jobs:
         with:
           channel: stable
           cache: true
-          cache-key: flutter3.00 #change this to force refresh cache
+          cache-key: flutter-ios-release-cache #change this to force refresh cache
```

Comment:
It should be `flutter3.0.1` 

As in https://github.com/openfoodfacts/smooth-app/blob/develop/.github/workflows/postsubmit.yml#L30

Reply:
do we have to have the same cache as postsubmit ? is there a downside to sharing the cache ?

Reply:
also, why did it start failing now while last commit is 16 days ago (16-6 should have been 10 days ago ?)